front/app/containers/Admin/projects/project/analysis/Demographics/AuthorsByAge.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from '@citizenlab/cl2-component-library';
 import { useParams } from 'react-router-dom';
 import {
-  BarChart,
+  BarChart as RechartsBarChart,
```

Comment:
Good solution indeed! Would be great if you could rename other occurrences in the app as well.